dhis-2/dhis-api/src/main/java/org/hisp/dhis/programrule/ProgramRuleActionType.java

```diff
@@ -40,17 +40,15 @@
  */
 public enum ProgramRuleActionType
 {
-    DISPLAYTEXT( "displaytext" ),
-    DISPLAYKEYVALUEPAIR( "displaykeyvaluepair" ),
```

Comment:
DisplayKeyValuePair is different from DisplayText, so they cant be replaced with showmessage directly without losing functionality.

Reply:
I was following the WHEN table in https://jira.dhis2.org/browse/DHIS2-7425

Should we change that to not group these 2 together into SHOWMESSAGE?

Reply:
Ok, thanks for pointing that out. We should not group them. Added a comment in the Jira to have Rodolfo update the spec.